### PR TITLE
fix: no --tag prerelease for private module

### PR DESCRIPTION
**Pull request by whyvez**
fixes: #294

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -440,6 +440,14 @@ describe('cli', function () {
       commit('feat: first commit')
       execCli('--prerelease alpha').stdout.should.include('--tag alpha')
     })
+
+    it('does not advises use of --tag prerelease for private modules', function () {
```
Comment (stevemao) [inline test.js]: Still need to fix this: "does not advise"

**Comment by stevemao** [comment]
Otherwise LGTM :)

File: test.js (hunk 1 above)
Comment (whyvez) [reply test.js]: I had updated this but forgot to add it in my amended commit. (strike 2) 😬 Just forced pushed.

**Comment by bcoe** [comment]
@whyvez thank you for this contribution, I'll roll it out with the security update 👍

**Comment by bcoe** [comment]
@whyvez thank you for the contribution! Please give this a try:

`npm i standard-version@5.0.0`

If you'd like to contribute more to this project over time, don't be shy about joining the Slack:

http://devtoolscommunity.herokuapp.com/

**Pull request merged**